src/components/organisms/UserProfileModal/UserProfileModal.tsx

```diff
@@ -136,30 +135,36 @@ export const UserProfileModal: React.FC<UserProfileModalProps> = ({
   );
 };
 
+/**
+ * @debt I believe this relates to Playa features, which are legacy code that will be removed soon
+ * @deprecated legacy tech debt related to Playa, soon to be removed
+ */
 const SuspectedLocation: React.FC<{
   user: WithId<User>;
   currentVenue: WithId<AnyVenue>;
 }> = ({ user, currentVenue }) => {
-  // @debt This will currently load all venues in firebase into memory.. not very efficient
-  useFirestoreConnect("venues");
-  const allVenues = useSelector(orderedVenuesSelector);
+  const { relatedVenues } = useRelatedVenues({
+    currentVenueId: currentVenue.id,
+  });
+
+  const { userLocation } = useWorldUserLocation(user.id);
+  const { lastSeenIn } = userLocation ?? {};
```

Comment:
Maybe you can change this to `const lastSeenIn = userLocation?.lastSeenIn` instead of destructuring an empty object.